testnets/holesky/QUICK_START.md

```diff
@@ -351,29 +351,43 @@ To get started with the configuration, copy the following preset file:
 cp ./presets/sidecar-delegations-preset.env.example bolt-sidecar.env
 ```
 
-This preset file will run the bolt-sidecar in "delegation" mode, which is the
-recommended way to set it up.
+This preset file will run the bolt-sidecar with a pre-authorized signer specified in the delegations file, as well
+as enable firewall delegation. This is the recommended way to run the sidecar.
 
 Fill the configuration excluding the "Signing options"
 section, which will be covered below. Remember to set the
-`BOLT_SIDECAR_COMMITMENT_PRIVATE_KEY` to the operator private key registered in
+`BOLT_SIDECAR_OPERATOR_PRIVATE_KEY` to the operator private key registered in
 the previous step.
 
-**Why delegation?**
+**Why firewall delegation?**
+Firewall delegation allows proposers to set an external third party as their network entrypoint or *firewall*. It gets rid of the
+requirement to expose an HTTP RPC endpoint for accepting inclusion requests (inbound), and instead subscribes to the configured firewall over a
+websocket connection (outbound). The firewall will then stream valid, filtered inclusion requests over the websocket connection.
+
+Some of the other duties of the firewall include:
+- Spam and DoS prevention
+- Pricing inclusion requests correctly (see more below)
+- Communicating prices with consumers (wallets, users)
+
+Currently, we operate a firewall RPC on Holesky at `wss://rpc-holesky.bolt.chainbound.io/api/v1/firewall_stream`.
```

Comment:
should we mention that we use http fallback if websocket fails?

Reply:
Can you elaborate more? You can't run both websocket and http server

Reply:
Wait, we do?

Reply:
I misunderstood